pkg/build/builder/cmd/dockercfg/cfg.go

```diff
@@ -49,7 +56,7 @@ func (h *Helper) GetDockerAuth(imageName, authType string) (docker.AuthConfigura
 	glog.V(3).Infof("Getting docker auth in paths : %v", searchPaths)
 	cfg, err = GetDockerConfig(searchPaths)
 	if err != nil {
-		glog.Errorf("Reading docker config from %v failed: %v", searchPaths, err)
+		glogreal.Errorf("Reading docker config from %v failed: %v", searchPaths, err)
```

Comment:
Why is this glogreal?

Reply:
because the glog wrapper doesn't implement "Errorf" and since the glog wrapper's would send anything at the "error" level to glog anyway, it was easier to just call real glog directly rather than add a pointless Errorf delegator to the glog wrapper.

Reply:
i may have gotten that backwards.

Reply:
in any case this was logging direct to glog before, and now it still will, but all the other glog logic is using the wrapper(where it previously was not)

Reply:
and since glog wrapper is only intended to pretty print info messages, i think sending an error to "real" glog is the right thing to do.